tests/integration/orcid/cassettes/TestOrcidPusherDuplicatedIdentifier.test_happy_flow.yaml

```diff
@@ -0,0 +1,313 @@
+interactions:
+- request:
+    body: null
+    headers:
+      Accept:
+      - application/orcid+json
+      Accept-Encoding:
+      - gzip, deflate
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '0'
+      Content-Type:
+      - application/orcid+json
+    method: DELETE
+    uri: https://api.orcid.org/v2.0/0000-0003-1134-6827/work/57442758
+  response:
+    body:
+      string: '{"response-code":404,"developer-message":"404 Not Found: The resource
```

Comment:
This seems wrong. As I said in a previous comment, you don't seem to be testing the case where you have duplicate identifiers, as the other paper is not there on ORCID at the beginning.